src/bundle/Resources/views/themes/admin/account/forgot_password/index.html.twig

```diff
@@ -3,7 +3,10 @@
 {% form_theme form_forgot_user_password '@ezdesign/account/form_fields.html.twig'  %}
 
 {%- block content -%}
-    <h2 class="ez-login__header">{{ 'authentication.reset_your_password'|trans|desc('Reset your password') }}</h2>
+    <h2 class="ez-login__actions-headline">{{ 'authentication.reset_your_password'|trans|desc('Reset your password') }}</h2>
+    {% if reason is defined and reason == userForgotPasswordReasonMigration %}
```

Comment:
So what is `userForgotPasswordReasonMigration`? is it variable or string cast from undefined variable? (just asking, not sure how Twig works in that regard and I don't see you setting it anywhere).

Reply:
https://github.com/ezsystems/ezplatform-user/pull/75/files#diff-25ac52efb0c1414c55ef4d9926512f1fR108

Reply:
To be honest, I'm wondering if we need this `reason is defined` here, I think we can keep this condision as simple as `reason == userForgotPasswordReasonMigration`, that do you think guys?

Reply:
changed to `reason == userForgotPasswordReasonMigration`